### PR TITLE
feat: add help class to display additional cmd help

### PR DESCRIPTION
**Pull request by peternhale**
@W-9791511@ add custom help class

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "state": "beta",
     "topicSeparator": " ",
     "commands": "./dist/commands",
+    "helpClass": "./dist/help/sfHelp.js",
```
Comment (peternhale) [inline package.json:32]: name the help class for oclif

File: src/help/sfCommandHelp.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { CommandHelp, HelpSection, HelpSectionRenderer, Interfaces } from '@oclif/core';
+import { SfCommandInterface } from '@salesforce/command';
+type SectionType = { header: string; generate: HelpSectionRenderer };
+
+export class SfCommandHelp extends CommandHelp {
```
Comment (peternhale) [inline src/help/sfCommandHelp.ts:11]: This class is responsible for collecting the help sections to display

File: src/help/sfCommandHelp.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  public constructor(
+    public command: Interfaces.Command,
+    public config: Interfaces.Config,
+    public opts: Interfaces.HelpOptions
+  ) {
+    super(command, config, opts);
+  }
+
+  protected sections(): SectionType[] {
```
Comment (peternhale) [inline src/help/sfCommandHelp.ts:20]: Override base class sections

File: src/help/sfCommandHelp.ts [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    const sections = super.sections();
+    const additionaSfSections: SectionType[] = [
```
Comment (peternhale) [inline src/help/sfCommandHelp.ts:22]: sf additional sections

File: src/help/sfHelp.ts [hunk 5]
```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { CommandHelp, Help } from '@oclif/core';
+import { SfCommandHelp } from './sfCommandHelp';
+
+export default class SfHelp extends Help {
```
Comment (peternhale) [inline src/help/sfHelp.ts:10]: Needed to specify SfCommandHelp as the class to call

**Review by mdonnalley — DISMISSED** [review]

**Review by RodEsp — DISMISSED** [review]

**Review by mdonnalley — APPROVED** [review]

**Pull request merged**